broker/network/network.go

```diff
@@ -55,13 +55,19 @@ func NewServer(c inter.Framework, bi BuildInfoProvider, debug bool) (s *Server,
 		mu:      &sync.Mutex{},
 	}
 
+	// Validate timeouts
+	if s.config.Choria.NetworkAuthTimeout > 60*time.Second || s.config.Choria.NetworkTLSTimeout > 60*time.Second {
```

Comment:
@ripienaar I have a version of this that uses the validator package instead, but when I looked in there I couldn't help but refactor it so that might to be a different PR

Reply:
Yeah validator is a bit long in the tooth for sure, be nice to get things to it for expressing limits like this.